crates/sui-framework/sources/collectible/collectible.move

```diff
@@ -158,6 +167,20 @@ module sui::collectible {
         res
     }
 
+    // === Borrows ===
```

Comment:
Is there a need to borrow the transfer policy cap as well?  I notice it's wrapped in a `Referent`.

Reply:
I will do it in the follow up along with royalties and commissions. Collectible should probably be reworked.